src/ripple/core/tests/Config.test.cpp

```diff
@@ -0,0 +1,356 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of rippled: https://github.com/ripple/rippled
+    Copyright (c) 2012-2015 Ripple Labs Inc.
+
+    Permission to use, copy, modify, and/or distribute this software for any
+    purpose  with  or without fee is hereby granted, provided that the above
+    copyright notice and this permission notice appear in all copies.
+
+    THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+    WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+    MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+    ANY  SPECIAL ,  DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+    WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+    ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+    OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+
+#include <BeastConfig.h>
+#include <ripple/core/impl/LoadFeeTrackImp.h>
+#include <ripple/core/Config.h>
+#include <ripple/basics/TestSuite.h>
+#include <boost/filesystem.hpp>
+#include <boost/format.hpp>
+#include <fstream>
+#include <iostream>
+
+namespace ripple {
+namespace detail {
+std::string configContents (std::string const& dbPath)
+{
+    static boost::format configContentsTemplate (R"rippleConfig(
```

Comment:
Ach, I knew about raw strings, I just never saw anyone use such a specific string for the delimiter before!  /me shakes head... carry on!